getFeatureFlagById/main.go

```diff
@@ -11,8 +11,7 @@ import (
 )
 
 func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
-	id, found := req.PathParameters["flagId"]
-
+	id, _ := req.PathParameters["flagId"]
```

Comment:
Removed this validation, as the lambda is invoked if we have the `flagId` being passed.